File: python/0001_Grading.py
a = int(input())
b = int(input())
c = int(input())
sum = a+b+c
if sum<=100 and sum>=80:
    print("A")
elif sum>=75 and sum<80:
    print("B+")
elif sum>=70 and sum<75:
    print("B")
elif sum>=65 and sum<70:
    print("C+")
elif sum>=60 and sum<65:
    print("C")
elif sum>=55 and sum<60:
    print("D+")
elif sum>=50 and sum<55:
    print("D")
elif sum>=0 and sum<50:
    print("F")

File: c/0001_Grading.c
#include <stdio.h>
int main(void)
{
    unsigned short int a, b, c, sum;
    scanf("%hu", &a);
    scanf("%hu", &b);
    scanf("%hu", &c);
    sum = a+b+c;
    if (sum>=80 && sum<=100)
        printf("A");
    else if (sum>=75 && sum<80)
        printf("B+");
    else if (sum>=70 && sum<75)
        printf("B");
    else if (sum>=65 && sum<70)
        printf("C+");
    else if (sum>=60 && sum<65)
        printf("C");
    else if (sum>=55 && sum<60)
        printf("D+");
    else if (sum>=50 && sum<55)
        printf("D");
    else if (sum>=0 && sum<50)
        printf("F");

    return 0;
}
